tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -559,7 +567,7 @@ export class WebGPUBackend extends KernelBackend {
     }
 
     if (env().get('WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE') as
```

Comment:
side thought: I think this is hit quite often. Would it help to cache this `env().get()` call instead of calling it every time?

Reply:
WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE may be changed by the user. So it seems that there is not a good place to cache it.

Reply:
Got it.